### ISSUE TITLE
icons become more pixelated as scale increases

### ISSUE THREAD

**Issue by Matronicz** [antonypro/QGoodWindow#31]
scale 100%
![image](https://user-images.githubusercontent.com/102349655/227522322-b97e8a42-8c79-4983-9c2a-f85305026974.png)
scale 125%
![image](https://user-images.githubusercontent.com/102349655/227522403-4a4a7d20-2304-4760-a218-f7bfbf500cbe.png)
scale 150%
![image](https://user-images.githubusercontent.com/102349655/227522416-b749ff20-691d-408c-abc3-774557a97121.png)
As you see, the icons becomes more pixelated.

I am using: Windows 11 with Qt 6.4.0

**Comment by antonypro** [comment]
@Majticz: Thanks for reporting.

This problem will be solved in next version.

**Comment by galvinhoang** [comment]
This may fix it. I used a QSVGWidget to directly display an svg file. Therefore it requires different svg files for white or black icons (depending on the window theme). I only found the maximize and restore icons in a Microsoft GitHub repository, so I quickly created the others myself. It might be possible to change the svg colors dynamically using QSvgWidget::load(const QByteArray &contents) and switching the colors using QByteArray functionalities.

[FIX.zip](https://github.com/antonypro/QGoodWindow/files/11668436/FIX.zip)
